pset-iterators.js

```diff
@@ -42,11 +42,13 @@ Expected Result Shape: A string representing the full name of the president that
 console.log('Problem 1:')
 
 // Add your code below this line
-
+const firstWhigPresident = presidents.filter(president => {
+	return president.party == "Whig";
+});
 // Add your code above this line
 
 /** added for formatting purposes **/
-console.log('')
+console.log(firstWhigPresident)
 console.log('-----------------')
```

Comment:
@jl1982 Good Job here on this solution. One suggestion I see you used the filter method which will return all of the presidents in the whig party but you only needed to return the first president in the whig party. Take a look at the `find` method in javascript [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find). The find() method returns the value of the first element in the provided array that satisfies the provided testing function.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L35)